smack-extensions/src/test/java/org/jivesoftware/smackx/VCardUnitTest.java

```diff
@@ -35,40 +37,222 @@ public void initSmackTestSuite() {
     }
 
     @Test
+    public void testParseFullVCardIQStanza() throws Throwable {
+
+        // @formatter:off
+        final String request =
+        "<iq id='v1' to='user@igniterealtime.org/mobile' type='result'>"
```

Comment:
Did it previously accept case insensitive vcards? If not, then I say we leave it as it is. It appears to me that XMPP is case sensitive just as XML is. And vcard is the only XEP I'm aware of that uses uppercase element names.